features/service-operation.feature

```diff
@@ -125,6 +125,19 @@ Feature: Command output from service operations like stop/start/restart
     Then the service "docker" should be running
     And have a new pid for "docker" service
 
+    When the "kubernetes" service is running
```

Comment:
There is are notes in the underlying step definitions (env.rb) that these steps are not working for Kubernetes. If they do the note should be removed